src/user/DOME_initialization.F90

```diff
@@ -396,14 +396,24 @@ subroutine DOME_set_Open_Bdry_Conds(OBC, tv, G, param_file, tr_Reg)
                                          log((2.0+Ri_trans)/(2.0-Ri_trans))
       do J=JsdB,JedB ; do i=isd,ied
         if (OBC_mask_v(i,J)) then
+           if (k > 3) then
```

Comment:
Likewise, I'm not including this change in the merge because it changes the DOME configuration and seems unrelated to the adding sponges in ALE mode.